options/linux/include/netpacket/packet.h

```diff
@@ -1,6 +1,8 @@
 #ifndef _NETPACKET_PACKET_H
 #define _NETPACKET_PACKET_H
 
+#define PACKET_HOST 0
```

Comment:
LGTM now that this PR is only a single line change :^)

Reply:
Isn't this an ABI thing?

Reply:
If it is, it matches Linux so it should be good.

Reply:
I think Arsen means that it should live in `abis/`